config/config.ini

```diff
@@ -0,0 +1,23 @@
+[hey]
+message1 = "おーきに"
+           "元気でっか？"
+           "呼んだか？"
+
+[add]
+error_message1 = "あかーん！もうリストがいっぱいや！最大 {} 個までしか追加できんで"
+
+[kimetaro]
+message1 = "よーうし、決めたるで〜"
+message2 = "むむっ、これや！"
+           "神のお告げに従って決めたったで"
+
+[list]
+message1 = "リストにあるのはこれやで"
+error_message1 = "残念やったな！リストはからっぽや！`/add "タスク"`でタスクを追加できるで"
+
+
+[remove]
+message1 = "登録されたリストは削除しといたで"
```

Comment:
It's better not to have a separate message. I'll fix it.

Reply:
What do you mean?

Reply:
I mean that it's better not to have the number suffix like message<integer> for maintainability.
Actually, these messages will be chosen at random, so we don't need to have a number suffix(or order)